Bakis/ClientApp/src/app/components/event-page/event-page.component.css

```diff
@@ -9,3 +9,17 @@ td.mat-cell:last-of-type, td.mat-footer-cell:last-of-type, th.mat-header-cell:la
 .top-of-page-actions{
     margin-left: 10px;
 }
+
+.switchButton {
+    background: #00AC9F;
+    color: white;
+}
+
+.actionButtons{
```

Comment:
if you are using this class in all tables maybe you could consider changing it to the global class